exporter.go

```diff
@@ -371,16 +371,22 @@ func (e *Exporter) entrypoint(launchMD launch.Metadata, defaultProcessType strin
 	if !e.supportsMulticallLauncher() {
 		return launch.LauncherPath, nil
 	}
-	if defaultProcessType == "" {
-		if len(launchMD.Processes) == 1 {
```

Comment:
I believe for older platform api, we still want to preserve the behavior of setting the only process as the default.

Reply:
Makes sense. I'll add it back.